APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1141,6 +1208,29 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 }
             }
                 break;
+                
+            case kAPCUserInfoItemTypeDateOfBirth:
+            {
+                if (self.canEditBirthDate &&
```

Comment:
I think that the method isEditingAllowedForHealthKitProperty was improperly names and some of the business logic of what I was doing right here leaked into that method.  So, I pushed the "canEdit" check into this part of the code instead.  Does that make more sense now?